csm/plugins/cortx/query_deployment.py

```diff
@@ -0,0 +1,492 @@
+# CORTX-CSM: CORTX Management web and CLI interface.
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+from cortx.utils.log import Log
+from csm.common.plugin import CsmPlugin
+from csm.common.certificate import SSLCertificate
+from csm.core.blogic import const
+#TODO: Uncomment after integration
+#from cortx.utils.query_deployment import QueryDeployment
+
+
+class QueryDeploymentPlugin(CsmPlugin):
+    """
+    Communicates with utils framework for fetching
+    cortx topology.
+    """
+    def __init__(self):
+        """
+        Initialize query deployment plugin
+        """
+        # set valid resources along with their attributes.
+        self.valid_resources = {'cluster':['id', 'version', 'nodes', 'storage_set', 'certificate']}
+        self.output = {
```

Comment:
What is self.output ? and its purpose ?

Reply:
Added just for testing feature branch quickly. will be removed in  integration

Reply:
please add this in schema dir, create a .json file and read that over here.

Reply:
The JSON is still there

Reply:
Removed json. It was added for integration testing.